### ISSUE TITLE
Cannot black list specific structures from The Beginning

### ISSUE THREAD

**Issue by Reshy** [TelepathicGrunt/RepurposedStructures#157]
Here's the provided config and screenshot below, yes the file type's changed so this can be uploaded.

![2021-11-20_06 28 50](https://user-images.githubusercontent.com/2096384/142730029-484ec0ec-291f-49a4-b886-ffc4b8e5851f.png)
[biome_dimension_allow_disallow_configs - Copy.txt](https://github.com/TelepathicGrunt/RepurposedStructures/files/7574667/biome_dimension_allow_disallow_configs.-.Copy.txt)

**Comment by TelepathicGrunt** [comment]
Actually it seems you typoed my modid. 
  "repursosed_structures:mineshaft_jungle": "thebeginning:theabyssdimbeginning",
  
It should be repurposed_structures instead of repursosed_structures. Please fix this typo and let me know if the config works

**Comment by Reshy** [comment]
I made the fix, but it still occurs.  The blocking all structures does block these from spawning, but not the jungle mineshafts specifically.  
[biome_dimension_allow_disallow_configs - Copy.txt](https://github.com/TelepathicGrunt/RepurposedStructures/files/7575924/biome_dimension_allow_disallow_configs.-.Copy.txt)

**Comment by TelepathicGrunt** [comment]
using your config, I see this error in the logs:

```

[21Nov2021 11:18:40.668] [Render thread/ERROR] [com.telepathicgrunt.repurposedstructures.configs.omegaconfig.OmegaConfig/]: com.google.gson.JsonSyntaxException: duplicate key: repurposed_structures:village_jungle
[21Nov2021 11:18:40.669] [Render thread/INFO] [com.telepathicgrunt.repurposedstructures.configs.omegaconfig.OmegaConfig/]: Encountered an error while reading repurposed_structures-forge/biome_dimension_allow_disallow_configs config, falling back to default values.
[21Nov2021 11:18:40.669] [Render thread/INFO] [com.telepathicgrunt.repurposedstructures.configs.omegaconfig.OmegaConfig/]: If this problem persists, delete the config file repurposed_structures-forge/biome_dimension_allow_disallow_configs.json5 and try again.
```

map can only have one key. Doing multiple keys breaks it (which can be a bit confusing yeah). That's why in the value section, instead of doing:
![image](https://user-images.githubusercontent.com/40846040/142770156-aed433b3-5095-4fb6-b69c-8fee7ed43a58.png)


do this instead:
![image](https://user-images.githubusercontent.com/40846040/142770185-ca3d3f1c-9da9-4bc7-b318-ce9116fc9f67.png)

Basically combine the two jungle village entries so the two dimensions are in the same entry by separated by a comma. After I fixed that config entry, it did not throw anymore errors and the mineshaft stopped spawning in that dimension.

Just curious, is there any RS structure that fit in that thebeginning's dimensions? I am thinking of adding `thebeginning:.+` to the "all" entry in my default config value to keep my structures from all of their dimensions. Wasn't sure if there was a structure that does fit in their dimension that I should be aware of

**Comment by Reshy** [comment]
Ah, so it cannot take in duplicate entries like that.  Makes more sense.  

There's two dimensions, one's in the sky, and one's on the ground, both are jungle themed.  The sky one the only one that seems to fit is the Fortress as the dimension is fairly barren (and jungle temples, assuming they don't spawn on the bottom of the world).  The groundlands is much more forgiving, everything but villages generates mostly alright, villages however sometimes have the roads cover the trees.

**Comment by TelepathicGrunt** [comment]
Thanks! I’ll close this now
